CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
+  - Remove start_date attribute
```

Comment:
Good question.
If we just "deprecate" something, I'd assume that it's still available and raise a waring.
If we "deprecate and remove" something, I'd assume it will raise a fatal error.
If we "remove" something, I'm just afraid it's not crystal clear that what we removed is a _feature_.

@MattiSG @michelbl @Anna-Livia thoughts ?